applications/sdp/mspi/src/main.c

```diff
@@ -23,7 +23,10 @@
 #define DATA_PINS_MAX 8
 #define VIO_COUNT     11
 
-#define MAX_FREQUENCY 64000000
+#define MAX_FREQUENCY		   64000000
+#define STD_PAD_BIAS_FREQ_TRESHOLD 32000000
+
+#define PAD_BIAS 1
```

Comment:
Please, change name of this define to something like `PAD_BIAD_VALUE` so that it was clear, that it is value that is supposed to be written. Right now it might be misunderstood as enable/disable define.